bundle/src/main/java/com/adobe/acs/commons/wcm/comparisons/impl/PageCompareDataLoaderImpl.java

```diff
@@ -0,0 +1,44 @@
+/*
+ *
+ *  * #%L
+ *  * ACS AEM Commons Bundle
+ *  * %%
+ *  * Copyright (C) 2016 Adobe
+ *  * %%
+ *  * Licensed under the Apache License, Version 2.0 (the "License");
+ *  * you may not use this file except in compliance with the License.
+ *  * You may obtain a copy of the License at
+ *  *
+ *  *      http://www.apache.org/licenses/LICENSE-2.0
+ *  *
+ *  * Unless required by applicable law or agreed to in writing, software
+ *  * distributed under the License is distributed on an "AS IS" BASIS,
+ *  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  * See the License for the specific language governing permissions and
+ *  * limitations under the License.
+ *  * #L%
+ *
+ */
+package com.adobe.acs.commons.wcm.comparisons.impl;
+
+import com.adobe.acs.commons.wcm.comparisons.PageCompareData;
+import com.adobe.acs.commons.wcm.comparisons.PageCompareDataLoader;
+import org.apache.felix.scr.annotations.Component;
+import org.apache.felix.scr.annotations.Properties;
+import org.apache.felix.scr.annotations.Service;
+import org.apache.sling.api.resource.Resource;
+
+import javax.jcr.RepositoryException;
+
+
+@Component(label = "ACS AEM Commons - One-to-one compare",
```

Comment:
We can live this as just `@Component` since there are not properties (thus the label, description, and metatype=true are not needed